lib/proxy.js

```diff
@@ -15,12 +15,15 @@ var
   routeCache = LRUCache({ max: 5000 }),
   safe = require('safetimeout'),
   letsencrypt = require('./letsencrypt.js'),
-  Promise = require('bluebird');
+  Promise = require('bluebird'),
+  bodyParser = require('body-parser');
 
```

Comment:
since the parser can be sent as an option, it is better to even force the user to depend on ```body-parser``` even if it is not interested in this feature.